etc/acme/templates/cert.ctmpl

```diff
@@ -1 +1,2 @@
-{{if key "nginx/acme/cert"}}{{key "nginx/acme/cert"}}{{end}}
+{{ $service_name := env "SERVICE_NAME" }}
```

Comment:
This appears fixed with `{{ $service_name := or $service_name "nginx" }}` below.